llvm/lib/Transforms/Yk/ControlPoint.cpp

```diff
@@ -118,85 +114,93 @@ class YkControlPoint : public ModulePass {
 
   bool runOnModule(Module &M) override {
     LLVMContext &Context = M.getContext();
+    std::vector<CallInst *> ControlPointCalls = findControlPointCalls(M);
```

Comment:
Might be an idea to assert that we don't discover more than two control point calls?

Recently I saw a scenario where the loop containing the control point got unrolled, and an assertion to catch that would be most welcome.

https://github.com/ykjit/yk/issues/1414

Reply:
sounds good.

Reply:
Added :point_right: 2291a0164970550aafbc4c1671c0a7b7df1b2c6a